.env

```diff
@@ -0,0 +1,10 @@
+MOCHA_PASSWORD=Ape1sOn4Hill!
```

Comment:
Passwords and secrets should not be checked in. Please remove this and change them as they are no longer secure. Any secrets that are needed will need to be loaded by CI securely.